# This file is part of HiGFXback

# requires
REQUIRES="autotools-wrappers-ac-build g++-build"

pkg-config --exists --print-errors $REQUIRES || exit 1

# configure (ac-2.64)
./configure --disable-libatomic --disable-libcilkrts --disable-libgomp --disable-libmpx --disable-libquadmath --disable-libsanitizer --disable-libssp --disable-libvtv --disable-lto --disable-multilib --disable-plugin --enable-languages=c++ --with-gxx-include-dir=/include/c++ --with-native-system-header-dir=/include --libexecdir=/lib --prefix=

# build
make

# install
install -d $DESTDIR/bin
install host-`./config.guess`/gcc/cpp $DESTDIR/bin
install host-`./config.guess`/gcc/xg++ $DESTDIR/bin/g++
ln -sf g++ $DESTDIR/bin/c++
install host-`./config.guess`/gcc/xgcc $DESTDIR/bin/gcc
ln -sf gcc $DESTDIR/bin/cc
install -d $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/c_global/cassert $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/c_global/cctype $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/c_global/cerrno $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/c_global/cfenv $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/c_global/cfloat $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/c_global/ciso646 $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/c_global/cinttypes $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/c_global/climits $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/c_global/clocale $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/c_global/cmath $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/c_global/csetjmp $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/c_global/csignal $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/c_global/cstdarg $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/c_global/cstddef $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/c_global/cstdint $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/c_global/cstdio $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/c_global/cstdlib $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/c_global/cstring $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/c_global/cwchar $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/c_global/ctime $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/c_global/cwctype $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/std/algorithm $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/std/array $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/std/atomic $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/std/bitset $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/std/chrono $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/std/complex $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/std/condition_variable $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/std/deque $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/std/forward_list $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/std/fstream $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/std/functional $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/std/future $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/std/iomanip $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/std/ios $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/std/iosfwd $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/std/iostream $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/std/istream $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/std/iterator $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/std/limits $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/std/list $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/std/locale $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/std/map $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/std/memory $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/std/mutex $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/std/numeric $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/std/ostream $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/std/queue $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/std/random $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/std/ratio $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/std/regex $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/std/set $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/std/shared_mutex $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/std/sstream $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/std/stack $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/std/stdexcept $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/std/streambuf $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/std/string $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/std/system_error $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/std/thread $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/std/tuple $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/std/type_traits $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/std/unordered_map $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/std/unordered_set $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/std/utility $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/std/valarray $DESTDIR/include/c++
install -m 644 libstdc++-v3/include/std/vector $DESTDIR/include/c++
install -d $DESTDIR/include/c++/backward
install -m 644 libstdc++-v3/include/backward/auto_ptr.h $DESTDIR/include/c++/backward
install -m 644 libstdc++-v3/include/backward/binders.h $DESTDIR/include/c++/backward
install -d $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/algorithmfwd.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/alloc_traits.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/allocated_ptr.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/allocator.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/atomic_base.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/atomic_futex.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/basic_ios.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/basic_ios.tcc $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/basic_string.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/basic_string.tcc $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/c++0x_warning.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/char_traits.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/codecvt.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/concept_check.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/cpp_type_traits.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/deque.tcc $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/forward_list.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/forward_list.tcc $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/fstream.tcc $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/functexcept.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/functional_hash.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/gslice.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/gslice_array.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/hashtable.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/hashtable_policy.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/indirect_array.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/ios_base.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/istream.tcc $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/list.tcc $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/locale_classes.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/locale_classes.tcc $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/locale_conv.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/locale_facets.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/locale_facets.tcc $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/locale_facets_nonio.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/locale_facets_nonio.tcc $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/localefwd.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/mask_array.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/memoryfwd.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/move.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/ostream.tcc $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/ostream_insert.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/parse_numbers.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/postypes.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/predefined_ops.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/ptr_traits.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/quoted_string.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/random.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/random.tcc $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/range_access.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/regex.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/regex.tcc $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/regex_automaton.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/regex_automaton.tcc $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/regex_compiler.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/regex_compiler.tcc $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/regex_constants.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/regex_error.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/regex_executor.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/regex_executor.tcc $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/regex_scanner.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/regex_scanner.tcc $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/shared_ptr.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/shared_ptr_atomic.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/shared_ptr_base.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/slice_array.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/sstream.tcc $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/std_mutex.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/stl_algo.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/stl_algobase.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/stl_bvector.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/stl_construct.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/stl_deque.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/stl_function.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/stl_heap.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/stl_iterator.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/stl_iterator_base_funcs.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/stl_iterator_base_types.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/stl_list.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/stl_map.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/stl_multimap.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/stl_multiset.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/stl_numeric.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/stl_pair.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/stl_queue.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/stl_raw_storage_iter.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/stl_relops.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/stl_set.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/stl_stack.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/stl_tempbuf.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/stl_tree.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/stl_uninitialized.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/stl_vector.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/stream_iterator.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/streambuf.tcc $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/streambuf_iterator.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/stringfwd.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/uniform_int_dist.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/unique_ptr.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/unordered_map.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/unordered_set.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/uses_allocator.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/valarray_after.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/valarray_array.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/valarray_array.tcc $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/valarray_before.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/include/bits/vector.tcc $DESTDIR/include/c++/bits
install -d $DESTDIR/include/c++/debug
install -m 644 libstdc++-v3/include/debug/assertions.h $DESTDIR/include/c++/debug
install -m 644 libstdc++-v3/include/debug/debug.h $DESTDIR/include/c++/debug
install -d $DESTDIR/include/c++/ext
install -m 644 libstdc++-v3/include/ext/aligned_buffer.h $DESTDIR/include/c++/ext
install -m 644 libstdc++-v3/include/ext/alloc_traits.h $DESTDIR/include/c++/ext
install -m 644 libstdc++-v3/include/ext/atomicity.h $DESTDIR/include/c++/ext
install -m 644 libstdc++-v3/include/ext/concurrence.h $DESTDIR/include/c++/ext
install -m 644 libstdc++-v3/include/ext/new_allocator.h $DESTDIR/include/c++/ext
install -m 644 libstdc++-v3/include/ext/numeric_traits.h $DESTDIR/include/c++/ext
install -m 644 libstdc++-v3/include/ext/string_conversions.h $DESTDIR/include/c++/ext
install -m 644 libstdc++-v3/include/ext/type_traits.h $DESTDIR/include/c++/ext
install -d $DESTDIR/include/c++/tr1
install -m 644 libstdc++-v3/include/tr1/memory $DESTDIR/include/c++/tr1
install -m 644 libstdc++-v3/include/tr1/shared_ptr.h $DESTDIR/include/c++/tr1
install -m 644 libstdc++-v3/include/tr1/tuple $DESTDIR/include/c++/tr1
install -m 644 libstdc++-v3/include/tr1/type_traits $DESTDIR/include/c++/tr1
install -m 644 libstdc++-v3/libsupc++/atomic_lockfree_defines.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/libsupc++/cxxabi.h $DESTDIR/include/c++
install -m 644 libstdc++-v3/libsupc++/cxxabi_forced.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/libsupc++/exception $DESTDIR/include/c++
install -m 644 libstdc++-v3/libsupc++/exception_defines.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/libsupc++/exception_ptr.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/libsupc++/hash_bytes.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/libsupc++/initializer_list $DESTDIR/include/c++
install -m 644 libstdc++-v3/libsupc++/nested_exception.h $DESTDIR/include/c++/bits
install -m 644 libstdc++-v3/libsupc++/new $DESTDIR/include/c++
install -m 644 libstdc++-v3/libsupc++/typeinfo $DESTDIR/include/c++
install -d $DESTDIR/include/c++/`./config.guess`/bits
install -m 644 `./config.guess`/libstdc++-v3/include/`./config.guess`/bits/atomic_word.h $DESTDIR/include/c++/`./config.guess`/bits
install -m 644 `./config.guess`/libstdc++-v3/include/`./config.guess`/bits/basic_file.h $DESTDIR/include/c++/`./config.guess`/bits
install -m 644 `./config.guess`/libstdc++-v3/include/`./config.guess`/bits/c++allocator.h $DESTDIR/include/c++/`./config.guess`/bits
install -m 644 `./config.guess`/libstdc++-v3/include/`./config.guess`/bits/c++config.h $DESTDIR/include/c++/`./config.guess`/bits
install -m 644 `./config.guess`/libstdc++-v3/include/`./config.guess`/bits/c++io.h $DESTDIR/include/c++/`./config.guess`/bits
install -m 644 `./config.guess`/libstdc++-v3/include/`./config.guess`/bits/c++locale.h $DESTDIR/include/c++/`./config.guess`/bits
install -m 644 `./config.guess`/libstdc++-v3/include/`./config.guess`/bits/cpu_defines.h $DESTDIR/include/c++/`./config.guess`/bits
install -m 644 `./config.guess`/libstdc++-v3/include/`./config.guess`/bits/ctype_base.h $DESTDIR/include/c++/`./config.guess`/bits
install -m 644 `./config.guess`/libstdc++-v3/include/`./config.guess`/bits/ctype_inline.h $DESTDIR/include/c++/`./config.guess`/bits
install -m 644 `./config.guess`/libstdc++-v3/include/`./config.guess`/bits/cxxabi_tweaks.h $DESTDIR/include/c++/`./config.guess`/bits
install -m 644 `./config.guess`/libstdc++-v3/include/`./config.guess`/bits/error_constants.h $DESTDIR/include/c++/`./config.guess`/bits
install -m 644 `./config.guess`/libstdc++-v3/include/`./config.guess`/bits/gthr-default.h $DESTDIR/include/c++/`./config.guess`/bits
install -m 644 `./config.guess`/libstdc++-v3/include/`./config.guess`/bits/gthr.h $DESTDIR/include/c++/`./config.guess`/bits
install -m 644 `./config.guess`/libstdc++-v3/include/`./config.guess`/bits/messages_members.h $DESTDIR/include/c++/`./config.guess`/bits
install -m 644 `./config.guess`/libstdc++-v3/include/`./config.guess`/bits/opt_random.h $DESTDIR/include/c++/`./config.guess`/bits
install -m 644 `./config.guess`/libstdc++-v3/include/`./config.guess`/bits/os_defines.h $DESTDIR/include/c++/`./config.guess`/bits
install -m 644 `./config.guess`/libstdc++-v3/include/`./config.guess`/bits/time_members.h $DESTDIR/include/c++/`./config.guess`/bits
install -d $DESTDIR/lib
ln -sf ../bin/cpp $DESTDIR/lib/cpp
install host-`./config.guess`/gcc/libgcc_s.so.1 $DESTDIR/lib
ln -sf libgcc_s.so.1 $DESTDIR/lib/libgcc_s.so
install `./config.guess`/libstdc++-v3/src/.libs/libstdc++.so.6.* $DESTDIR/lib/libstdc++.so.6
ln -sf libstdc++.so.6 $DESTDIR/lib/libstdc++.so
install -d $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`
install host-`./config.guess`/gcc/cc1 $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`
install host-`./config.guess`/gcc/cc1plus $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`
install -m 644 host-`./config.guess`/gcc/crtbegin.o $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`
install -m 644 host-`./config.guess`/gcc/crtbeginS.o $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`
install -m 644 host-`./config.guess`/gcc/crtend.o $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`
install -m 644 host-`./config.guess`/gcc/crtendS.o $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`
install -m 644 host-`./config.guess`/gcc/crtfastmath.o $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`
install -m 644 host-`./config.guess`/gcc/libgcc.a $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`
install -d $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/adxintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/ammintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/avxintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/avx2intrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/avx512bwintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/avx512cdintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/avx512dqintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/avx512erintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/avx512fintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/avx512ifmaintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/avx512ifmavlintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/avx512pfintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/avx512vbmiintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/avx512vbmivlintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/avx512vlbwintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/avx512vldqintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/avx512vlintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/bmiintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/bmi2intrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/clflushoptintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/clwbintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/clzerointrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/cpuid.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/emmintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/f16cintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/float.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/fma4intrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/fmaintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/fxsrintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/ia32intrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/immintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/lwpintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/lzcntintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/mm3dnow.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/mm_malloc.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/mmintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/mwaitxintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/nmmintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/pkuintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/pmmintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/popcntintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/prfchwintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/rdseedintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/rtmintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/shaintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/smmintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/stdarg.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/stdatomic.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/stdbool.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/stddef.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/stdnoreturn.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/tbmintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/tmmintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/unwind.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/wmmintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/x86intrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/xmmintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/xopintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/xsavecintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/xsaveintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/xsaveoptintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/xsavesintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/xtestintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -d $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include-fixed
install -m 644 host-`./config.guess`/gcc/include-fixed/limits.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include-fixed

# build.pc
install -d $DESTDIR/share/pkgconfig
cat > $DESTDIR/share/pkgconfig/libstdc++-build.pc << EOF
Name: gcc
Version: 6.5.0
Description: GNU Compiler Collection
Requires: $REQUIRES

devel=\\
/bin/c++ \\
/bin/cc \\
/bin/cpp \\
/bin/g++ \\
/bin/gcc \\
/include/c++/algorithm \\
/include/c++/array \\
/include/c++/atomic \\
/include/c++/bitset \\
/include/c++/cassert \\
/include/c++/cctype \\
/include/c++/cerrno \\
/include/c++/cfenv \\
/include/c++/cfloat \\
/include/c++/chrono \\
/include/c++/ciso646 \\
/include/c++/cinttypes \\
/include/c++/climits \\
/include/c++/clocale \\
/include/c++/cmath \\
/include/c++/condition_variable \\
/include/c++/complex \\
/include/c++/csetjmp \\
/include/c++/csignal \\
/include/c++/cstdarg \\
/include/c++/cstddef \\
/include/c++/cstdint \\
/include/c++/cstdio \\
/include/c++/cstdlib \\
/include/c++/cstring \\
/include/c++/ctime \\
/include/c++/cwchar \\
/include/c++/cwctype \\
/include/c++/cxxabi.h \\
/include/c++/deque \\
/include/c++/exception \\
/include/c++/forward_list \\
/include/c++/fstream \\
/include/c++/functional \\
/include/c++/future \\
/include/c++/initializer_list \\
/include/c++/iomanip \\
/include/c++/ios \\
/include/c++/iosfwd \\
/include/c++/iostream \\
/include/c++/istream \\
/include/c++/iterator \\
/include/c++/limits \\
/include/c++/list \\
/include/c++/locale \\
/include/c++/map \\
/include/c++/memory \\
/include/c++/mutex \\
/include/c++/new \\
/include/c++/numeric \\
/include/c++/ostream \\
/include/c++/queue \\
/include/c++/random \\
/include/c++/ratio \\
/include/c++/regex \\
/include/c++/set \\
/include/c++/shared_mutex \\
/include/c++/sstream \\
/include/c++/stack \\
/include/c++/stdexcept \\
/include/c++/streambuf \\
/include/c++/string \\
/include/c++/system_error \\
/include/c++/thread \\
/include/c++/tuple \\
/include/c++/type_traits \\
/include/c++/typeinfo \\
/include/c++/unordered_map \\
/include/c++/unordered_set \\
/include/c++/utility \\
/include/c++/valarray \\
/include/c++/vector \\
/include/c++/backward/auto_ptr.h \\
/include/c++/backward/binders.h \\
/include/c++/bits/algorithmfwd.h \\
/include/c++/bits/alloc_traits.h \\
/include/c++/bits/allocated_ptr.h \\
/include/c++/bits/allocator.h \\
/include/c++/bits/atomic_base.h \\
/include/c++/bits/atomic_futex.h \\
/include/c++/bits/atomic_lockfree_defines.h \\
/include/c++/bits/basic_ios.h \\
/include/c++/bits/basic_ios.tcc \\
/include/c++/bits/basic_string.h \\
/include/c++/bits/basic_string.tcc \\
/include/c++/bits/c++0x_warning.h \\
/include/c++/bits/char_traits.h \\
/include/c++/bits/codecvt.h \\
/include/c++/bits/concept_check.h \\
/include/c++/bits/cpp_type_traits.h \\
/include/c++/bits/cxxabi_forced.h \\
/include/c++/bits/cxxabi_tweaks.h \\
/include/c++/bits/deque.tcc \\
/include/c++/bits/exception_defines.h \\
/include/c++/bits/exception_ptr.h \\
/include/c++/bits/forward_list.h \\
/include/c++/bits/forward_list.tcc \\
/include/c++/bits/fstream.tcc \\
/include/c++/bits/functexcept.h \\
/include/c++/bits/functional_hash.h \\
/include/c++/bits/gslice.h \\
/include/c++/bits/gslice_array.h \\
/include/c++/bits/hash_bytes.h \\
/include/c++/bits/hashtable.h \\
/include/c++/bits/hashtable_policy.h \\
/include/c++/bits/indirect_array.h \\
/include/c++/bits/ios_base.h \\
/include/c++/bits/istream.tcc \\
/include/c++/bits/list.tcc \\
/include/c++/bits/locale_classes.h \\
/include/c++/bits/locale_classes.tcc \\
/include/c++/bits/locale_conv.h \\
/include/c++/bits/locale_facets.h \\
/include/c++/bits/locale_facets.tcc \\
/include/c++/bits/locale_facets_nonio.h \\
/include/c++/bits/locale_facets_nonio.tcc \\
/include/c++/bits/localefwd.h \\
/include/c++/bits/mask_array.h \\
/include/c++/bits/memoryfwd.h \\
/include/c++/bits/move.h \\
/include/c++/bits/nested_exception.h \\
/include/c++/bits/ostream.tcc \\
/include/c++/bits/ostream_insert.h \\
/include/c++/bits/parse_numbers.h \\
/include/c++/bits/postypes.h \\
/include/c++/bits/predefined_ops.h \\
/include/c++/bits/ptr_traits.h \\
/include/c++/bits/quoted_string.h \\
/include/c++/bits/random.h \\
/include/c++/bits/random.tcc \\
/include/c++/bits/range_access.h \\
/include/c++/bits/regex.h \\
/include/c++/bits/regex.tcc \\
/include/c++/bits/regex_automaton.h \\
/include/c++/bits/regex_automaton.tcc \\
/include/c++/bits/regex_compiler.h \\
/include/c++/bits/regex_compiler.tcc \\
/include/c++/bits/regex_constants.h \\
/include/c++/bits/regex_error.h \\
/include/c++/bits/regex_executor.h \\
/include/c++/bits/regex_executor.tcc \\
/include/c++/bits/regex_scanner.h \\
/include/c++/bits/regex_scanner.tcc \\
/include/c++/bits/shared_ptr.h \\
/include/c++/bits/shared_ptr_atomic.h \\
/include/c++/bits/shared_ptr_base.h \\
/include/c++/bits/slice_array.h \\
/include/c++/bits/sstream.tcc \\
/include/c++/bits/std_mutex.h \\
/include/c++/bits/stl_algo.h \\
/include/c++/bits/stl_algobase.h \\
/include/c++/bits/stl_bvector.h \\
/include/c++/bits/stl_construct.h \\
/include/c++/bits/stl_deque.h \\
/include/c++/bits/stl_function.h \\
/include/c++/bits/stl_heap.h \\
/include/c++/bits/stl_iterator.h \\
/include/c++/bits/stl_iterator_base_funcs.h \\
/include/c++/bits/stl_iterator_base_types.h \\
/include/c++/bits/stl_list.h \\
/include/c++/bits/stl_map.h \\
/include/c++/bits/stl_multimap.h \\
/include/c++/bits/stl_multiset.h \\
/include/c++/bits/stl_numeric.h \\
/include/c++/bits/stl_pair.h \\
/include/c++/bits/stl_queue.h \\
/include/c++/bits/stl_raw_storage_iter.h \\
/include/c++/bits/stl_relops.h \\
/include/c++/bits/stl_set.h \\
/include/c++/bits/stl_stack.h \\
/include/c++/bits/stl_tempbuf.h \\
/include/c++/bits/stl_tree.h \\
/include/c++/bits/stl_uninitialized.h \\
/include/c++/bits/stl_vector.h \\
/include/c++/bits/stream_iterator.h \\
/include/c++/bits/streambuf.tcc \\
/include/c++/bits/streambuf_iterator.h \\
/include/c++/bits/stringfwd.h \\
/include/c++/bits/uniform_int_dist.h \\
/include/c++/bits/unique_ptr.h \\
/include/c++/bits/unordered_map.h \\
/include/c++/bits/unordered_set.h \\
/include/c++/bits/uses_allocator.h \\
/include/c++/bits/valarray_after.h \\
/include/c++/bits/valarray_array.h \\
/include/c++/bits/valarray_array.tcc \\
/include/c++/bits/valarray_before.h \\
/include/c++/bits/vector.tcc \\
/include/c++/debug/assertions.h \\
/include/c++/debug/debug.h \\
/include/c++/ext/aligned_buffer.h \\
/include/c++/ext/alloc_traits.h \\
/include/c++/ext/atomicity.h \\
/include/c++/ext/concurrence.h \\
/include/c++/ext/new_allocator.h \\
/include/c++/ext/numeric_traits.h \\
/include/c++/ext/string_conversions.h \\
/include/c++/ext/type_traits.h \\
/include/c++/tr1/memory \\
/include/c++/tr1/shared_ptr.h \\
/include/c++/tr1/tuple \\
/include/c++/tr1/type_traits \\
/include/c++/`./config.guess`/bits/atomic_word.h \\
/include/c++/`./config.guess`/bits/basic_file.h \\
/include/c++/`./config.guess`/bits/c++allocator.h \\
/include/c++/`./config.guess`/bits/c++config.h \\
/include/c++/`./config.guess`/bits/c++io.h \\
/include/c++/`./config.guess`/bits/c++locale.h \\
/include/c++/`./config.guess`/bits/cpu_defines.h \\
/include/c++/`./config.guess`/bits/ctype_base.h \\
/include/c++/`./config.guess`/bits/ctype_inline.h \\
/include/c++/`./config.guess`/bits/error_constants.h \\
/include/c++/`./config.guess`/bits/gthr-default.h \\
/include/c++/`./config.guess`/bits/gthr.h \\
/include/c++/`./config.guess`/bits/messages_members.h \\
/include/c++/`./config.guess`/bits/opt_random.h \\
/include/c++/`./config.guess`/bits/os_defines.h \\
/include/c++/`./config.guess`/bits/time_members.h \\
/lib/cpp \\
/lib/libgcc_s.so \\
/lib/libstdc++.so \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/cc1 \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/cc1plus \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/crtbegin.o \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/crtbeginS.o \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/crtend.o \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/crtendS.o \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/crtfastmath.o \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/libgcc.a \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/adxintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/ammintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/avxintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/avx2intrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/avx512bwintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/avx512cdintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/avx512dqintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/avx512erintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/avx512fintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/avx512ifmaintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/avx512ifmavlintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/avx512pfintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/avx512vbmiintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/avx512vbmivlintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/avx512vlbwintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/avx512vldqintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/avx512vlintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/bmiintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/bmi2intrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/clflushoptintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/clwbintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/clzerointrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/cpuid.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/emmintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/f16cintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/float.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/fma4intrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/fmaintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/fxsrintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/ia32intrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/immintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/lwpintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/lzcntintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/mm3dnow.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/mm_malloc.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/mmintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/mwaitxintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/nmmintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/pkuintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/pmmintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/popcntintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/prfchwintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/rdseedintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/rtmintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/shaintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/smmintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/stdarg.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/stdatomic.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/stdbool.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/stddef.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/stdnoreturn.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/tbmintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/tmmintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/unwind.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/wmmintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/x86intrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/xmmintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/xopintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/xsavecintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/xsaveintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/xsaveoptintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/xsavesintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/xtestintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include-fixed/limits.h

exec=\\
/lib/libgcc_s.so.1 \\
/lib/libstdc++.so.6
EOF

exit

# patch
--- gcc-6.5.0.orig/gcc/config/i386/linux64.h
+++ gcc-6.5.0/gcc/config/i386/linux64.h
@@ -28,8 +28,8 @@
 #define GNU_USER_LINK_EMULATIONX32 "elf32_x86_64"
 
 #define GLIBC_DYNAMIC_LINKER32 "/lib/ld-linux.so.2"
-#define GLIBC_DYNAMIC_LINKER64 "/lib64/ld-linux-x86-64.so.2"
-#define GLIBC_DYNAMIC_LINKERX32 "/libx32/ld-linux-x32.so.2"
+#define GLIBC_DYNAMIC_LINKER64 "/lib/ld-linux-x86-64.so.2"
+#define GLIBC_DYNAMIC_LINKERX32 "/lib/ld-linux-x32.so.2"
 
 #undef MUSL_DYNAMIC_LINKER32
 #define MUSL_DYNAMIC_LINKER32 "/lib/ld-musl-i386.so.1"
--- gcc-6.5.0.orig/gcc/config/i386/t-linux64
+++ gcc-6.5.0/gcc/config/i386/t-linux64
@@ -33,6 +33,6 @@
 comma=,
 MULTILIB_OPTIONS    = $(subst $(comma),/,$(TM_MULTILIB_CONFIG))
 MULTILIB_DIRNAMES   = $(patsubst m%, %, $(subst /, ,$(MULTILIB_OPTIONS)))
-MULTILIB_OSDIRNAMES = m64=../lib64$(call if_multiarch,:x86_64-linux-gnu)
+MULTILIB_OSDIRNAMES = m64=../lib$(call if_multiarch,:x86_64-linux-gnu)
 MULTILIB_OSDIRNAMES+= m32=$(if $(wildcard $(shell echo $(SYSTEM_HEADER_DIR))/../../usr/lib32),../lib32,../lib)$(call if_multiarch,:i386-linux-gnu)
 MULTILIB_OSDIRNAMES+= mx32=../libx32$(call if_multiarch,:x86_64-linux-gnux32)

# source
https://gcc.gnu.org/pub/gcc/releases/gcc-6.5.0/gcc-6.5.0.tar.xz
